sql_generators/glean_usage/common.py

```diff
@@ -234,22 +234,23 @@ def generate_per_app_id(
                     **render_kwargs,
                 )
 
+        # generated files to update
+        Artifact = namedtuple("Artifact", "table_id basename sql")
+        artifacts = [
+            Artifact(view, "metadata.yaml", view_metadata),
+            Artifact(table, "metadata.yaml", table_metadata),
+            Artifact(table, "query.sql", query_sql),
+        ]
+
         if not (referenced_table_exists(view_sql)):
             logging.info("Skipping view for table which doesn't exist:" f" {table}")
-            return
```

Comment:
This also skips generating the query of the table that is referenced in the view in the first place

Reply:
This must have worked at some point? But it hasn't for the past two years? curious...